Cannot find module '@babel/runtime-corejs3/core-js/object/define-property'

Hey guys, we are getting an error after deployment: 

`Cannot find module '@babel/runtime-corejs3/core-js/object/define-property' Require stack: - /var/task/src/node_modules/@redwoodjs/api/dist/index.js - /var/task/src/api/dist/functions/graphql.js - /var/task/s_graphQL.js - /var/runtime/UserFunction.js - /var/runtime/index.js`



Some hints:
- It is not happening in the local environment just after deploying to AWS.  
- It is fixed if we downgrade to rw version 0.26.0
- It seems the core-js module is not being included in /api/dist/node_modules
- We have a custom deployment flow using zip-it-and-ship-it and serverless.yml. We have a serverless.yml inside /api and after running a zip-it-and-ship-it we include the artifact like this:

```
graphQL:
    handler: src/api/dist/functions/graphql.handler
    package:
      artifact: zipball/graphql.zip
    ...etc
```

Any insight would be great, thanks a lot! :) 

Leo

our package.json root:

```
{
  "private": true,
  "workspaces": {
    "packages": [
      "api",
      "web",
      "integrations/*"
    ]
  },
  "devDependencies": {
    "@redwoodjs/core": "^0.35.2",
    "babel-plugin-module-resolver": "^4.0.0",
    "eslint-config-airbnb": "^18.2.1",
    "eslint-plugin-react-hooks": "^4.2.0",
    "git-last-commit": "^1.0.0",
    "minimist": "^1.2.5",
    "serverless-prune-plugin": "^1.4.3"
  },
  "eslintConfig": {
    "extends": "@redwoodjs/eslint-config"
  },
  "engines": {
    "node": "14.x",
    "yarn": ">=1.15"
  },
  "scripts": {
    "wmx-push": "node ./scripts/push_and_deploy",
    "web-server": "cd web && yarn run webpack-dev-server --config ../node_modules/@redwoodjs/core/config/webpack.development.js",
    "api-server": "cd api && yarn run dev-server",
    "build-web-sls": "yarn rw build web && node ./scripts/createIndexHtml.js"
  }
}
```

our package.json API:
```

{
  "name": "api",
  "version": "0.0.0",
  "private": true,
  "dependencies": {
    "@redwoodjs/api": "^0.35.2",
    "@sentry/node": "^5.20.0",
    "activecampaign": "^1.2.5",
    "aws-sdk": "^2.725.0",
    "cuid": "^2.1.8",
    "dayjs": "^1.10.1",
    "exchange-rates-api": "^1.0.2",
    "fetch-retry": "^4.0.0",
    "js-yaml": "^4.0.0",
    "lodash.flatten": "^4.4.0",
    "lodash.uniq": "^4.5.0",
    "moment": "^2.26.0",
    "node-fetch": "^2.6.0",
    "qs": "^6.9.4",
    "serverless": "^2.11.0",
    "serverless-layers": "^2.2.4",
    "serverless-webpack": "^5.3.3",
    "stripe": "^8.64.0",
    "webpack-node-externals": "^2.5.0",
    "yargs": "^15.4.1",
    "@babel/runtime": "^7.14.8",
    "@babel/runtime-corejs3": "^7.14.9"
  },
  "devDependencies": {
    "@netlify/zip-it-and-ship-it": "^2.1.3",
    "chalk": "^4.1.0",
    "serverless-dotenv-plugin": "^3.0.0",
    "serverless-layers": "^2.2.4",
    "serverless-plugin-ifelse": "^1.0.5",
    "serverless-prune-plugin": "^1.4.3"
  }
}
```

--- a/babel.config.js
+++ b/babel.config.js
@@ -53,18 +53,6 @@ module.exports = {
     ['@babel/plugin-proposal-decorators', { legacy: true }],
     ['@babel/plugin-proposal-class-properties', { loose: true }],
     ['@babel/plugin-proposal-private-methods', { loose: true }],
-    [
-      '@babel/plugin-transform-runtime',
-      {
-        // https://babeljs.io/docs/en/babel-plugin-transform-runtime/#core-js-aliasing
-        // Setting the version here also requires `@babel/runtime-corejs3`
-        corejs: { version: 3, proposals: true },
-        // https://babeljs.io/docs/en/babel-plugin-transform-runtime/#version
-        // Transform-runtime assumes that @babel/runtime@7.0.0 is installed.
-        // Specifying the version can result in a smaller bundle size.
-        version: packageJSON.devDependencies['@babel/runtime-corejs3'],
-      },
-    ],
   ],
   overrides: [
     // ** WEB PACKAGES **

--- a/babel.config.js
+++ b/babel.config.js
@@ -91,6 +91,18 @@ module.exports = {
             ],
           },
         ],
+        [
+          '@babel/plugin-transform-runtime',
+          {
+            // https://babeljs.io/docs/en/babel-plugin-transform-runtime/#core-js-aliasing
+            // Setting the version here also requires `@babel/runtime-corejs3`
+            corejs: { version: 3, proposals: true },
+            // https://babeljs.io/docs/en/babel-plugin-transform-runtime/#version
+            // Transform-runtime assumes that @babel/runtime@7.0.0 is installed.
+            // Specifying the version can result in a smaller bundle size.
+            version: packageJSON.devDependencies['@babel/runtime-corejs3'],
+          },
+        ],
         // normally provided through preset-env detecting TARGET_BROWSER
         // but webpack 4 has an issue with this
         // see https://github.com/PaulLeCam/react-leaflet/issues/883

--- a/packages/api/package.json
+++ b/packages/api/package.json
@@ -10,7 +10,6 @@
   "types": "./dist/index.d.ts",
   "license": "MIT",
   "dependencies": {
-    "@babel/runtime": "7.15.3",
     "@graphql-tools/merge": "7.0.0",
     "@prisma/client": "2.29.1",
     "@types/pino": "6.3.11",

